src/lib/logger/formatter/index.ts

```diff
@@ -11,7 +11,6 @@ module.exports = function prettyFactory(options: PrettyOptionsExtended): PrettyF
   // the break line must happens in the prettify component
   const breakLike = '\n';
   return (inputData): string => {
-    // FIXME: review colors by default is true
-    return printMessage(inputData, options, true) + breakLike;
+    return printMessage(inputData, options, options.colors) + breakLike;
```

Comment:
I'd like to limit this feature to a environment variable, allowing this property will hit users on migration since I'm not sure if I want to move this to 6.x and they will ask why is not working.  Let's use env variable only for now.

Reply:
well. this calls for a redo...

can you share your considerations for why it will not move on to the next version?

The use-case is - running verdaccio as an intermediate registry as part of a multi-stage build.
Each stage builds some artifacts, run their tests and publishes to the intermediate registry.
Once all stages have passed - all artifacts are published to the corporate registry.

To troubleshoot we collect the artifactory logs as build artifacts, which gitlab sadly won't present, but just let us download them as files. As such - we want them timestamped-pretty, and with no TTL color characters.

Reply:
⌛ back soon to this, got busy this week 😓

Reply:
> can you share your considerations for why it will not move on to the next version?

I dind't see your ticket https://github.com/verdaccio/verdaccio/issues/3007 , I understand you want to get rid of the `�[33m200�` .

My reason is I want to move logging output to plugins, but well, this is ok.